content/applications/marketing/survey.rst

```diff
@@ -1,8 +1,266 @@
+  is allowed
+
+* **Single Line Text Box** - open-text field suitable for short answers where any type of text is
+  allowed
+
+* **Numerical Value** - open-text field that only accepts a number as an answer
+
+* **Date** - date selection on a calendar
+
+* **Datetime** - date and time selection on a calendar
+
+* **Multiple choice: only one answer** - closed question where only one answer can be selected
+
+* **Multiple choice: multiple answers allowed** - closed question where multiple answers can be
+  selected
+
+* **Matrix** - table of multiple choice questions that share the same possible answers
+
+Answers and Options tab
+-----------------------
+
+Features change in the *Answers* and *Options* tab. It all depends on the *Question Type*.
+
+.. note::
+   Some options are common to many *Question Types*, such as *Validate entry*, *Mandatory
+   Answer*, *Question Time Limit* (for *Live Sessions*), *Conditional Display*, and *Score/Scored*.
+
+Input must be an email
+~~~~~~~~~~~~~~~~~~~~~~
+
+Force participants to enter an email address by ticking *Input must be an email*. That way, you
+can continue to communicate with them, even after they've completed the survey.
+
+.. note::
+   Compatible *Question Type*: *Single Line Text Box*
+
+Save as user nickname
+~~~~~~~~~~~~~~~~~~~~~
+
+Tick *Save as user nickname* to save their response as their username. This username is needed in
+order for them to login next time they visit the website.
+
+.. note::
+   Compatible *Question Type*: *Single Line Text Box*
+
+Validate entry
+~~~~~~~~~~~~~~
+
+Entry validation lets you define requirements for single line text length, dates, and numerical
+values. To do so, go to the *Answers* tab, tick *Validate entry*, and set a *Minimum* and a
+*Maximum* value.
+
+You can also customize the *Validation Error message* that appears beneath the field on the survey
+if an invalid answer is entered.
+
+.. note::
+   Compatible *Question Types*: *Single Line Text Box*, *Numerical Value*, *Date*, *Datetime*
+
+Conditional Display
+~~~~~~~~~~~~~~~~~~~
+
+*Conditional Display* means a question is only displayed if the specified conditional answer has
+been selected in a previous question.
+
+To use that feature, go to a question's *Options* tab, activate *Conditional Display*, choose a
+*Triggering Question* and select the *Triggering Answer* linked to that question. Remember,
+*Triggering Questions* can only be multiple choice.
+
+.. image:: survey/conditional-triggering-answers.png
+   :align: center
+   :alt: Conditional triggering answer fields
+
+.. tip::
+   Save the entire survey first before selecting a triggering question to update the selection of
+   triggering questions.
+
+.. note::
+   Compatible with **all** *Question Types*
+
+Question Time Limit (for *Live Sessions*)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Timed questions are only available during *Live Session* surveys.
```

Comment:
```suggestion
Timed questions are only available during :ref:`survey/live-session` surveys.
```
Here you can add a `:ref:` as explained in my first comment. You will need to add the target to the Live Session section as I did further down in my review.